frontend/src/theme/components.scss

```diff
@@ -54,3 +53,9 @@ h4 {
 .el-button--primary {
   box-shadow: 2px 3px 7px 0px $color-grey-7;
 }
+
+.LinkSide,
```

Comment:
You are declaring a variable, using the value of another variable. Why? Your variable is already declared ($spacing-s), you can just take usage in your components.

Another point. These classes should be in the View, not here